pkgs/top-level/all-packages.nix

```diff
@@ -27968,4 +27968,6 @@ in
 
   unifi-poller = callPackage ../servers/monitoring/unifi-poller {};
 
+  melody = callPackage ../applications/audio/melody {};
+
```

Comment:
This typically wouldn't get put at the end, but would probably be sorted into semi-alphabetical order in the range of lines 20000+ (don't have the file in front of me, so rough range)